lib/builder.js

```diff
@@ -93,6 +93,35 @@ class MessageBuilder {
         return this;
     }
 
+    /**
+     * Set the embedded url
+     * @param {string} url The url 
```

Comment:
Could use a nicer doc comment:
```suggestion
	 * Set the URL for the Discord embed
     *
     * @param {string} url The URL to link to from the embed.
```